packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/permissions/PermissionsModule.java

```diff
@@ -172,8 +172,14 @@ public void invoke(Object... args) {
   public boolean onRequestPermissionsResult(
       int requestCode, String[] permissions, int[] grantResults) {
     try {
-      mCallbacks.get(requestCode).invoke(grantResults, getPermissionAwareActivity());
-      mCallbacks.remove(requestCode);
+      Callback callback = mCallbacks.get(requestCode);
+      if (callback != null) {
+        callback.invoke(grantResults, getPermissionAwareActivity());
+        mCallbacks.remove(requestCode);
+      } else {
+        String message = "Unable to find callback with requestCode:" + requestCode;
+        FLog.w("PermissionsModule", message);
```

Comment:
nit: inline message
```suggestion
        FLog.w("PermissionsModule", "Unable to find callback with requestCode:" + requestCode);
```

Reply:
Actually, since this is FLog, you can do `FLog.w("PermissionsModule", "Unable to find callback with requestCode %d", requestCode)`.

Reply:
Thank you updated